test/test-sessions.js

```diff
@@ -72,4 +76,44 @@ describe('sessions', function () {
         });
     });
   });
+
+  describe.only("#pluckSession()", function () {
+    it("should pluck desired fsd session", function () {
+      console.log(dummySessions);
```

Comment:
@Wyndelle i should have mentioned, but try to make a habit of not pushing up logs for review. Its no an issue if you are pairing back and forth, but treat every review as if its ready for production @LoganVoisin @wilsuriel03